src/Manager.php

```diff
@@ -220,6 +221,7 @@ public function exportTranslations($group = null, $json = false)
             }
 
             Translation::ofTranslatedGroup(self::JSON_GROUP)->update(array('status' => Translation::STATUS_SAVED));
+			event(new TranslationsWasPublishedEvent($group));
```

Comment:
The `event()` is the Laravel's native method to dispatch events:
https://laravel.com/docs/5.3/events#firing-events